Assets/Scripts/ShadowPuzzle/PilarCheck.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.Experimental.GraphView;
+using UnityEngine;
+
+public class PilarCheck : MonoBehaviour
```

Comment:
```suggestion
public sealed class Pillar : MonoBehaviour
```
Miss een betere naam